### ISSUE TITLE
Enable Continuous capture of Satellite images

### ISSUE THREAD

**Issue by fccoelho** [AlertaDengue/AlertaDengueCaptura#50]
Continuing from the  work done in PR #42 , We need to automate the weekly capture of images through the creation of dedicated celery task which will run weekly. 
The images downloaded should be stored in the filesystem.

Once this is done a new issue can be opened to implement the postprocessing of the images to generate the data to be inserted in the table `Clima_Satelite`. This table schema may need to be redesigned, to conform with the data available. Issue #22 has an older proposal.

**Comment by esloch** [comment]
O service de celery terá de ser separado para crawlclima e downloader_app, cada um irá conter um compose por serparado para o docker.

**Comment by fccoelho** [comment]
[Fonte alternativa](https://cds.climate.copernicus.eu/toolbox/doc/gallery/54_box_plots.html) para captura das séries temporais de clima. @luabida can you take a look at this?

**Comment by fccoelho** [comment]
[Fonte alternativa](https://cds.climate.copernicus.eu/toolbox/doc/gallery/54_box_plots.html) para captura das séries temporais de clima. @luabida can you take a look at this?
O que precisamos é capturar séries de temperatura, humidade, chuva, etc. no entorno das [sedes dos municípios](https://www.ibge.gov.br/geociencias/organizacao-do-territorio/estrutura-territorial/27385-localidades.html?=&t=acesso-ao-produto).

**Comment by fccoelho** [comment]
[este exemplo](https://cds.climate.copernicus.eu/toolbox/doc/gallery/3_extract_time_series_and_plot_graph.html) já extrai as séries temporais
